### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by lisdarr** [Mckysse/MSD#2]
Hi, I would like to know when you train the model, do you use all sets of the source language (including training set, development set and test set), or only the training set of the source language? Looking forward to your answer!

**Comment by Mckysse** [comment]
When conducting parallel domain adaptation, we only use the training set. There several strategies to select source domain data and target domain data, like random matching.

**Comment by Mckysse** [comment]
When conducting parallel domain adaptation, we only use the training set. There are several strategies to select source domain data and target domain data, like random matching.
